helm/aws-operator/values.yaml

```diff
@@ -7,9 +7,12 @@ flag:
       labelselector:
         version: ""
 image:
-  registry: "quay.io"
   name: "giantswarm/aws-operator"
   tag: "[[ .Version ]]"
+Installation:
+  V1:
+    Registry:
+      Domain: quay.io
```

Comment:
Is that needed? The default is set here https://github.com/giantswarm/installations/blob/91d748669d9523f53d285f3cafde1812d7165261/default-draughtsman-configmap-values.yaml#L718

Reply:
Yes, for e2e test this is needed as the e2e installations tools dont use draightsman and dont have its defaults

Reply:
Understood. Then we should unify the approach with https://github.com/giantswarm/net-exporter/pull/77/files

Reply:
The problem is cert-exporter and net-exporter are deployed by both draughtsman and app CRs. So `image.registry` is needed for app CRs.

aws-operator is deployed by app CR but for VOO so only the draughtsman values are used. So I think `.Values.Installation.V1.Registry.Domain` is correct.

I find this confusing and maybe supporting both structures is better and more future proof?

Reply:
Yes, what @rossf7  said. I did that already to few places. I woudl leave it that way.

Reply:
👍